build_tools/embed_data/BUILD

```diff
@@ -32,41 +32,6 @@ cc_binary(
     ],
 )
 
-cc_embed_data(
-    name = "testembed1",
-    # do not sort
-    srcs = [
-        "file1.bin",
-        "data/file2.bin",
-    ],
-    cc_file_output = "testembed1.cc",
-    cpp_namespace = "foobar",
-    flatten = True,
-    h_file_output = "testembed1.h",
-)
-
-cc_embed_data(
-    name = "testembed2",
-    srcs = [
-        "data/file3.bin",
-    ],
-    cc_file_output = "testembed2.cc",
-    cpp_namespace = "foobar",
-    flatten = True,
-    h_file_output = "testembed2.h",
-)
-
-cc_test(
-    name = "cc_embed_data_test",
-    srcs = ["cc_embed_data_test.cc"],
-    deps = [
-        ":testembed1",
-        ":testembed2",
-        "//iree/testing:gtest",
-        "//iree/testing:gtest_main",
-    ],
-)
-
 c_embed_data(
```

Comment:
The c_embed_data_test has a smaller coverage than the cc_embed_data_test. We should extend the test coverage of c_embed_ data_test if cc_embed_data_test is removed.

Reply:
Good catch, done (and removed the old `_test.cc`)